release/models/policy-forwarding/openconfig-pf-srte.yang

```diff
@@ -124,6 +154,13 @@ module openconfig-pf-srte {
         protocol, or interface to the device, or from other static
         entries configured on the system.";
     }
+
+    leaf explicit-null-label {
+      type explicit-null-label-type;
+      description
+        "Indicates type of explicit null label that is being pushed on 
+        an unlabeled IP packet"
```

Comment:
(nit) Please add a full stop at the end of this sentence.

Reply:
done